homeassistant/components/switch/__init__.py

```diff
@@ -27,11 +27,16 @@
 
 MIN_TIME_BETWEEN_SCANS = timedelta(seconds=10)
 
+DISCOVER_CHILD_SWITCHES = 'switch.child_switches'
+DISCOVER_VERA_SWITCHES = 'switch.vera_switches'
```

Comment:
This should probably be in the Vera component right now like the other components.